core-web/libs/block-editor/src/lib/components/dot-block-editor/dot-block-editor.component.ts

```diff
@@ -133,7 +133,11 @@ export class DotBlockEditorComponent implements OnInit, OnDestroy {
     ) {}
 
     async loadCustomBlocks(urls: string[]) {
-        return Promise.all(urls.map(async (url) => import(/* webpackIgnore: true */ url)));
+        return Promise.all(
```

Comment:
We need to change this and make it more robust with `Promise.allSettle` method:

> Promise.all() method rejects itself if any of the passed in promise input inside an array is rejected. That is, this method will only run if and only if all the promises are fulfilled or resolved successfully, otherwise, in the output, it would produce an error message.
Promise.allSettled() method will not reject itself if any of the passed in promise input inside an array is rejected. That is, this method will irrespective of any promise be in the rejected state too.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled